test/contrib/icmp_extensions.uts

```diff
@@ -0,0 +1,7 @@
++ ICMP Extensions tests
+
+= Basic build
+
+p = IP(dst="142.250.209.14")/ICMP()/ICMPExtensionHeader(version = 2)/ICMPExtensionMPLS(classnum = 1, classtype = 1)
+b = b'E\x00\x00$\x00\x01\x00\x00@\x01/\xfc\xc0\xa8*+\x8e\xfa\xd1\x0e\x08\x00\xf6\xfa\x00\x00\x00\x00 \x00\xdf\xff\x00\x04\x01\x01'
```

Comment:
This value was built on a computer with 192.168.42.43 address to reach at least 142.250.209.14.
The test should force the source address to fix the problem:
```suggestion
p = IP(src="142.250.209.1", dst="142.250.209.14")/ICMP()/ICMPExtensionHeader(version = 2)/ICMPExtensionMPLS(classnum = 1, classtype = 1)
b = b'E\x00\x00$\x00\x01\x00\x00@\x01\xba\xd3\x8e\xfa\xd1\x01\x8e\xfa\xd1\x0e\x08\x00\xf6\xfa\x00\x00\x00\x00 \x00\xdf\xff\x00\x04\x01\x01'
```

Reply:
Done!